supervisor/daemon/transport/transport_linux.go

```diff
@@ -17,10 +17,56 @@
 
 package transport
 
-import "errors"
+import (
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+)
+
+const sock = "/var/run/myst.sock"
 
 // Start starts a listener on a unix domain socket.
 // Conversation is handled by the handlerFunc.
-func Start(handle handlerFunc, _ Options) error {
-	return errors.New("not implemented")
+func Start(handle handlerFunc, options Options) error {
+	if err := os.RemoveAll(sock); err != nil {
+		return fmt.Errorf("could not remove sock: %w", err)
+	}
+	l, err := net.Listen("unix", sock)
+	if err != nil {
+		return fmt.Errorf("error listening: %w", err)
+	}
+	numUid, err := strconv.Atoi(options.Uid)
+	if err != nil {
+		return fmt.Errorf("failed to parse uid %s: %w", options.Uid, err)
+	}
+	if err := os.Chown(sock, numUid, -1); err != nil {
+		return fmt.Errorf("failed to chown supervisor socket to uid %s: %w", options.Uid, err)
+	}
+	if err := os.Chmod(sock, 0700); err != nil {
```

Comment:
I am not sure what permissions should be applied here, but desktop app does not start unless it's `766` I believe

Reply:
As I understand desktop app is running in a different user space, so ye `0700` would not work.